e2e/specs/window-json-rpc-test/json-rpc-coverage.spec.js

```diff
@@ -0,0 +1,120 @@
+'use strict';
+import { web } from 'detox';
+import testCoverage from '@open-rpc/test-coverage';
+import { parseOpenRPCDocument } from '@open-rpc/schema-utils-js';
+import JsonSchemaFakerRule from '@open-rpc/test-coverage/build/rules/json-schema-faker-rule';
+import HtmlReporter from '@open-rpc/test-coverage/build/reporters/html-reporter';
+import TestHelpers from '../../helpers';
+import { SmokeCore } from '../../tags';
+import Browser from '../../pages/Browser';
+import TabBarComponent from '../../pages/TabBarComponent';
+import FixtureBuilder from '../../fixtures/fixture-builder';
+import { withFixtures } from '../../fixtures/fixture-helper';
+import { loginToApp } from '../../viewHelper';
+import ConnectedAccountsModal from '../../pages/modals/ConnectedAccountsModal';
+import { TestDApp } from '../../pages/TestDApp';
+import ConnectModal from '../../pages/modals/ConnectModal';
+
+import Assertions from '../../utils/Assertions';
+
+describe(SmokeCore(''), () => {
+  beforeAll(async () => {
+    jest.setTimeout(150000);
+    await TestHelpers.reverseServerPort();
+  });
+
+  it('', async () => {
+    await withFixtures(
+      {
+        dapp: true,
+        fixture: new FixtureBuilder().build(),
+        restartDevice: true,
+      },
+      async () => {
+        const openrpcDocument = await parseOpenRPCDocument(
+          'https://metamask.github.io/api-specs/latest/openrpc.json',
+        );
+
+        await loginToApp();
+        await TabBarComponent.tapBrowser();
+        await Browser.isVisible();
+        await Browser.navigateToTestDApp();
+
+        const webElement = await web.element(by.web.id('json-rpc-response'));
+        await webElement.scrollToView();
+
+        const pollResult = async () => {
+          let result = await webElement.runScript(`(el) => {
+            return el.innerHTML
+          }`);
+
+          while (result.result === '') {
+            await TestHelpers.delay(500);
+            result = await webElement.runScript(`(el) => {
+              return el.innerHTML
+            }`);
+          }
+
+          // await web.element(by.web.id('json-rpc-response')).replaceText('');
+
+          return result;
```

Comment:
probably also need to JSON.parse here
```suggestion
          return JSON.parse(result);
```